src/modules/display.js

```diff
@@ -0,0 +1,108 @@
+const taskContainer = document.querySelector('.movie-container');
+const involvementAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+const appID = 'fvEG8bcfusuKIAC9Au4g';
```

Comment:
- [ ] Kindly avoid code repetition here as well, these constants had already been declared earlier; implementing this will make your codebase cleaner. 👍 I think you can implement this in a .js file and export to wherever needed. ℹ️